.travis.yml

```diff
@@ -7,7 +7,7 @@ env:
   global:
     - secure: "LLqhKxqgRMp/C/TzZWv8YuhpmEm1twggm76NBUAQfZmOPLCkQSpAO8hoBM3qaIlDPSKPgoYj9f0TBuNi0iIFghQf0Xc4pXPCV0AnoGpXwRGiJATTAXfnG7RBa/hXRRBeAKlGmAI9GLtIoCQbUKYhq8gqwbzQVQXq+90rhsMH4zo="
     - CRATE_TESTS_SQL_REQUEST_TIMEOUT="20"
-    - _JAVA_OPTIONS="-Xmx1g"
+    - _JAVA_OPTIONS="-Xms512m -Xmx1024m"
```

Comment:
any reason to not set Xms and Xmx to the same value? we recommend to do that and we should do ourself as well ;)